jekyll/_cci2/version-control-system-integration-overview.adoc

```diff
@@ -141,6 +141,15 @@ include::../_includes/partials/tips/check-github-type.adoc[Check your GitHub int
 | [.circle-red]#**No**#
 | [.circle-green]#**Yes**#
 
+| Trigger pipelines on non-push events
```

Comment:
Could we make this a link to https://circleci.com/docs/github-trigger-event-options/ or maybe https://circleci.com/docs/triggers-overview/#run-a-pipeline-on-commit-to-your-code-repository ?

Reply:
Since the line is for all integrations, shouldn't we link to a page that is more general?